hushline/model.py

```diff
@@ -39,6 +49,9 @@ class User(Model):
     secondary_usernames: Mapped[Set["SecondaryUsername"]] = relationship(
         backref=db.backref("primary_user", lazy=True)
     )
+    smtp_encryption: Mapped[SMTPEncryption] = mapped_column(
+        db.Enum(SMTPEncryption, native_enum=False), default=SMTPEncryption.StartTLS
```

Comment:
Specifying `native_enum` as False here directs SQLalchemy to represent the enum as a string value instead of a native enum type. My reasoning here is that native enums require a db migration in order support new values, and it's not clear that you can easily add values to the enum.